Makefile

```diff
@@ -26,6 +26,9 @@ coverage: ## create coverage report
 	poetry run coverage run --source=src -m pytest
 	poetry run coverage xml
 
+build: ## create the wheel
```

Comment:
@fbraza Could you please move this to around line 46-67. This section is about testing and linting. And there are things related to packaging and submitting in there

Reply:
Done @vikramaditya91